Ticket - 43153 Enhancements to 1099 for tax year 2019

sewa67 opened the pull request:
Please review this change and merge into development branch 
Files - viewers/vend.w,windows/l-codedscr.w,aprep/r-1099m.i,aprep/r-1099m.w,aprep/r-v1099A.w,
             aprep/r-v1099N.w

sewa67 committed:
Ticket - 43153 Enhancements to 1099 for tax year 2019
Files - viewers/vend.w,windows/l-codedscr.w,aprep/r-1099m.i,aprep/r-1099m.w,aprep/r-v1099A.w,
             aprep/r-v1099N.w

sewa67 committed:
Ticket - 43153 Enhancements to 1099 for tax year 2019
Files - aprep/fibremsc.p

brad-advantzware reviewed (changes requested):
@sewa67 The list of 1,2,3 etc, should be centralized so that the # and value only exists in one place in code.  All items can be a LOOKUP function and all uses will be consistent.  Suggest comma separated variables for gc1099Code and gc1099CodeDesc.  Simplest is an include file

sewa67 committed:
Ticket - 43153 Enhancements to 1099 for tax year 2019
Files -  viewers/vend.w,aprep/r-1099m.i,aprep/r-1099m.w,aprep/r-v1099A.w,
             aprep/r-v1099N.w,sys/inc/boxcode1099.i
